backend/fms_core/migrations/0026_v3_5_0.py

```diff
@@ -0,0 +1,223 @@
+                                                            {'$id': 'fms:experimental_group',
+                                                             '$schema': 'http://json-schema.org/draft-07/schema#',
+                                                             'description': 'Schema used to define experimental groups for a sample.',
+                                                             'items': {'minLength': 1, 'type': 'string'},
+                                                             'title': 'Experimental group schema', 'type': 'array',
+                                                             'uniqueItems': True}, formats=None)])),
+                ('tissue_source', models.CharField(blank=True,
+                                                   choices=[('BAL', 'BAL'), ('Biopsy', 'Biopsy'), ('Blood', 'Blood'),
+                                                            ('Cells', 'Cells'), ('Expectoration', 'Expectoration'),
+                                                            ('Gargle', 'Gargle'), ('Plasma', 'Plasma'),
+                                                            ('Saliva', 'Saliva'), ('Swab', 'Swab'), ('Tumor', 'Tumor'),
+                                                            ('Buffy coat', 'Buffy coat'), ('Tail', 'Tail')],
+                                                   help_text='Can only be specified if the biospecimen type is DNA or RNA.',
+                                                   max_length=200)),
+                ('library', models.CharField(blank=True, max_length=200, null=True)),
+                ('index', models.CharField(blank=True, max_length=200, null=True)),
+                ('biosample', models.ForeignKey(help_text='Biosample associated to this DerivedSample',
+                                                on_delete=django.db.models.deletion.PROTECT,
+                                                related_name='derived_samples', to='fms_core.biosample')),
+                ('created_by', models.ForeignKey(blank=True, on_delete=django.db.models.deletion.PROTECT,
+                                                 related_name='fms_core_derivedsample_creation',
+                                                 to=settings.AUTH_USER_MODEL)),
+                ('sample_kind', models.ForeignKey(
+                    help_text='Biological material collected from study subject during the conduct of a genomic study project.',
+                    on_delete=django.db.models.deletion.PROTECT, to='fms_core.samplekind')),
+                ('updated_by', models.ForeignKey(blank=True, on_delete=django.db.models.deletion.PROTECT,
+                                                 related_name='fms_core_derivedsample_modification',
+                                                 to=settings.AUTH_USER_MODEL)),
+                ('sample', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.PROTECT,
```

Comment:
Did we say 'samples' was ManyToMany or it should stay as a ForeignKey 'sample'?

Reply:
I think this is a temporary field used during the migration ...